src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,114 @@
+.reset-margin {
+  margin: 0;
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
```

Comment:
box-sizing should be the same for all elements, use * selector for that and move to normalize